### PR TITLE
[cassandra staging ] update workloads API version to apps/v1 in k8s v1.9.0

### PR DESCRIPTION
**Pull request by Davidchinacloud**
Signed-off-by: LinWengang <linwengang@chinacloud.com.cn>

### CODE REVIEW

**Comment by ahmetb** [comment]
/lgtm

May I ask if you're doing these updates manually? If it's automated, why not do it for the entire repo?

**Comment by Davidchinacloud** [comment]
It's updates manually.

**Comment by Davidchinacloud** [comment]
Well, I'll try it.

File: cassandra/cassandra-statefulset.yaml [hunk 1]
```diff
@@ -1,10 +1,15 @@
-apiVersion: "apps/v1beta1"
+apiVersion: "apps/v1" # for versions before 1.9.0 use apps/v1beta2  and before 1.8.0 use apps/v1beta1
 kind: StatefulSet
 metadata:
   name: cassandra
+  labels:
```
Comment (ahmetb) [inline cassandra/cassandra-statefulset.yaml]: I think adding this label is not necessary as part of apps/v1 migration. Do other PRs made to other Kubernetes repos also add this label? If not, let’s not add labels.

Comment (Davidchinacloud) [reply cassandra/cassandra-statefulset.yaml]: It is recommended to add this lable in kubernetes.io examples  links： https://kubernetes.io/docs/tutorials/stateful-application/cassandra/

Comment (ahmetb) [reply cassandra/cassandra-statefulset.yaml]: That's fair, it should be here in the first place.

**Pull request merged**